### PR TITLE
Use SI suffixes instead of GNU suffixes for memory sizes

### PR DESCRIPTION
**Pull request by romasku**
Switches from GNU suffixes to SI suffixes. More information [here](https://en.wikipedia.org/wiki/Binary_prefix). In short:
Suffixes like **M** or **G** should be used for decimal quantifiers and **Mi** or **Gi** (pronounced Mebi and Gibi) should be used for binary quantifiers.

### CODE REVIEW

File: tests/cli/formatters/test_storage_formatters.py [hunk 1]
```diff
@@ -415,11 +415,11 @@ def test_long_formatter(self) -> None:
 
         formatter = LongFilesFormatter(human_readable=True, color=False)
         assert list(formatter(self.files_and_folders)) == [
-            "-r    2.0K 2018-01-01 03:00:00 File1",
-            "-r    1.0K 2018-10-10 13:10:10 File2",
-            "-r 1000.0K 2019-02-02 05:02:02 File3 with space",
-            "dm       0 2017-03-03 06:03:03 Folder1",
-            "dm       0 2017-03-03 06:03:02 1Folder with space",
+            "-r    2.0 Ki 2018-01-01 03:00:00 File1",
```
Comment (serhiy-storchaka) [inline tests/cli/formatters/test_storage_formatters.py:418]: Why does it show "Ki" and not "KiB" as for disks formatter?

**Review by serhiy-storchaka — COMMENTED** [review]
I am not sure that we should do it right now.

Advantages:
* More compatible with Kubernetes.
* Conforming to standards.

Disadvantages:
* Longer output (`"5 GiB"` vs `"5G"`, `"123 Bytes"` vs `"123B"`).
* Less compatible with GNU tools.

File: tests/cli/formatters/test_storage_formatters.py (hunk 1 above)
Comment (romasku) [reply tests/cli/formatters/test_storage_formatters.py]: It was striped in formatter (here is how it looks like on master brunch):
https://github.com/neuromation/platform-client-python/blob/18ecc3cf577a16c44df41fd24f14335e7cf10dc0/neuromation/cli/formatters/storage.py#L477-L478
Now I fixed that.

File: neuromation/cli/parse_utils.py [hunk 2]
```diff
@@ -9,46 +9,44 @@
...
+    }
+    result: Optional[int] = None
+    try:
+        result = int(memory)
+    except ValueError:
+        for suffixes, factor in suffix_to_factor.items():
+            if result:
+                break
+            for suffix in suffixes:
+                if memory.endswith(suffix):
```
Comment (asvetlov) [inline neuromation/cli/parse_utils.py:41]: From the practical point of view, I suggest supporting case-insensitive comparison. `MiB`, `MIB`, and `mib` are equal, isn't it?

**Pull request closed without merging**